normandy/selfrepair/static/js/index.js

```diff
@@ -0,0 +1,3 @@
+import 'babel-polyfill'
+import './normandy_driver.js'
+import './self_repair_runner.js'
```

Comment:
Why import `./normandy_driver.js` here? Wouldn't it get included through `self_repair_runner.js`?